src/lib/helpers/prune.ts

```diff
@@ -2,7 +2,7 @@ export function pruneObject<T>(obj: T) {
   const prunedObject = {} as T;
   Object.keys(obj).forEach((key) => {
     if (obj[key] != null && obj[key] != '' && obj[key] != []) {
-      if (obj[key] instanceof Object && !(obj[key] instanceof Array)) {
+      if (obj[key] instanceof Object && !(obj[key] instanceof Array) && key != 'coordinates') {
```

Comment:
@jacobbowdoin I merged this because I considered this was a hotfix. But please check if you agree with my change